ledger/participant-integration-api/src/main/scala/platform/store/backend/oracle/OracleResetStorageBackend.scala

```diff
@@ -27,6 +27,8 @@ object OracleResetStorageBackend extends ResetStorageBackend {
       "participant_users",
       "participant_user_rights",
       "transaction_metering",
-    ).map(table => SQL"truncate table #$table cascade").foreach(_.execute()(connection))
+    ) foreach { table =>
+      SQL"delete from #$table".execute()(connection)
+    }
```

Comment:
I am surprised delete is faster than truncate :) is it because of cascade?

Reply:
me too, oracle never gets boring ;)
(cascade is needed in our case)